ui/src/components/albumGallery/AlbumGallery.js

```diff
@@ -56,7 +56,9 @@ const AlbumGallery = ({
 
   useEffect(() => {
     const updateImageState = event => {
-      setImageState(event.state.imageState)
+      if (event.state.imageState) {
```

Comment:
Is there a reason you put this check here?
It seems to break fullscreen history functionallity.

When I visit an album, click full screen on an image and the press the back button, it doesn't go out of full screen, it does though if I comment this change out.

Reply:
Solved in later commit